crate/operator/main.py

```diff
@@ -114,6 +115,7 @@ async def cluster_create(
 
 @kopf.on.update(API_GROUP, "v1", RESOURCE_CRATEDB, id=CLUSTER_UPDATE_ID)
 @crate.on.error(error_handler=crate.send_update_failed_notification)
+@crate.timeout(timeout=float(config.ROLLING_RESTART_TIMEOUT))
```

Comment:
I am a bit uneasy about setting this to `ROLLING_RESTART_TIMEOUT` as it is set to 1 hour - which might not be enough in cases where we are, i.e., waiting for a backup to finish. But we can probably go with this and see how much of a problem it is

Reply:
We decided to add a new config option for the entire update process and set it to 2 hours for now.